projects/plugins/protect/src/js/state/actions.js

```diff
@@ -366,6 +368,14 @@ const setWafIsSeen = isSeen => {
 	return { type: SET_WAF_IS_SEEN, isSeen };
 };
 
+const setWafUpgradeIsSeen = upgradeIsSeen => {
```

Comment:
Looks like some actions from the work on the post-upgrade messaging snuck into this branch, and should probably be removed.